api/base/serializers.py

```diff
@@ -279,6 +299,53 @@ def to_representation(self, data):
             self.child.to_representation(item, envelope=None) for item in data
         ]
 
+    # Overrides ListSerializer which doesn't support multiple update by default
+    def update(self, instance, validated_data):
+        if len(instance) != len(validated_data):
+            raise exceptions.NotFound()
+        instance_mapping = {item._id: item for item in instance}
```

Comment:
After the end of the pass I'll sit down to look at this further with you- I'd like to be sure I understand why this is in the function before we discuss options. Inherently, it's a little weird because we're trying to make all of these functions inherently cover two separate sets of behavior (single vs bulk) at once.

In particular it seems odd to me that the serializer isn't handling things involving the ID field, as it normally does elsewhere.